### ISSUE TITLE
Getting "x509: certificate signed by unknown authority" errors when using gazette/brokers:v0.82.2

### ISSUE THREAD

**Issue by elynnyap** [gazette/core#227]
While trying to upgrade our broker cluster's version from `liveramp/gazette:2.0.210` to `gazette/brokers:v0.82.2`, our brokers starting producing these warning-level logs:

```
{"err":"Get https://www.googleapis.com/storage/v1/b/<redacted>: oauth2: cannot fetch token: Post https://oauth2.googleapis.com/token: x509: certificate signed by unknown authority","interval":300000000000,"level":"warning","msg":"failed to refresh remote fragments (will retry)","name":"us/pixel-serving/reporting/easy-alpaca/recovery/extractor/90dfb38b24c939ec8fea26cc","time":"2019-10-30T20:28:49Z"} 
```

```
{"err":"context canceled","interval":300000000000,"level":"warning","msg":"failed to refresh remote fragments (will retry)","name":"us/pixel-serving/reporting/easy-alpaca/recovery/vtable/c6f6f3afff7ff992901bd05e","time":"2019-10-31T16:12:09Z"} 
```

This appears to be because `ca-certificates` has not been installed in the new image:

```
$ docker run --rm -it gazette/broker:v0.82.2
gazette@2b515d8f7f67:~$ ls /etc/ssl
ls: cannot access '/etc/ssl': No such file or directory
gazette@2b515d8f7f67:~$ ls /usr/local/share 
man
```

versus the older image:

```
$ docker run --rm -it liveramp/gazette:2.0.210
root@c59f39eb094d:/go# ls /etc/ssl
certs  openssl.cnf  private
root@c59f39eb094d:/go# ls /usr/local/share
ca-certificates  man
```

**Comment by jgraettinger** [comment]
Thanks @elynnyap . This is a whoops, I got too KonMari while stripping down the release image to it's essentials. The problem is that `ca-certificates` was dropped and shouldn't have been. As we worked out over slack, and quick work-around is to kustomize in a patch:

```yaml
@@ -105,6 +105,8 @@ spec:
         volumeMounts:
         - name: tmpdir
           mountPath: /tmp
+        - name: certs
+          mountPath: /etc/ssl/certs
         - name: zonemap
           readOnly: true
           mountPath: /home/gazette/.config/zonemap
@@ -120,6 +122,10 @@ spec:
         hostPath:
           path: /mnt/disks/ssd0/tmp
           type: DirectoryOrCreate
+      - name: certs
+        hostPath:
+          path: /etc/ssl/certs
+          type: DirectoryOrCreate
```

**Comment by jgraettinger** [comment]
Fixed in `v0.84.2`
